integration/container/cdi_test.go

```diff
@@ -27,8 +27,12 @@ func TestCreateWithCDIDevices(t *testing.T) {
 
 	cwd, err := os.Getwd()
 	assert.NilError(t, err)
-	d := daemon.New(t, daemon.WithExperimental())
-	d.StartWithBusybox(ctx, t, "--cdi-spec-dir="+filepath.Join(cwd, "testdata", "cdi"))
+	configPath := filepath.Join(cwd, "daemon.json")
+	err = os.WriteFile(configPath, []byte(`{"features": {"cdi": true}}`), 0o644)
+	defer os.Remove(configPath)
+	assert.NilError(t, err)
+	d := daemon.New(t)
```

Comment:
This reminds me we should look into having a way to enable features with the command line arguments 🙈

Reply:
Yeah, I started with a new "option" to set these (for our tests), but then decided not to widen the scope of this PR too much.

ISTR not adding a flag was a decision made at the time (but may have been because features could be either `enabled` or `disabled`, so we would have to create a more complicated flag parsing to handle with that, so we decided to just limit it to configuration-file.